src/project_creator/templates/project/{{cookiecutter.name}}/Pipfile

```diff
@@ -4,25 +4,30 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
```

Comment:
Copy all dependencies from SPLED to get the documentation and integration tests working